vagrant/__init__.py

```diff
@@ -876,6 +876,13 @@ def plugin_list(self):
         output = self._run_vagrant_command(["plugin", "list", "--machine-readable"])
         return self._parse_plugin_list(output)
 
+    def validate(self):
+        """
+        This command validates present Vagrantfile.
+        """
+        output = self._run_vagrant_command(["validate"])
+        return output
```

Comment:
hm. I'm relying on the exit code to detect if it's valid or not. Since you're returning the output, does it mean that the exit code is not reliable ?

Reply:
For reference: https://github.com/ansible-community/molecule-vagrant/blob/0bc79f497c4ddfcde0336b308f0585fb320b13b9/molecule_vagrant/modules/vagrant.py#L560

Reply:
my bad, will use error codes

Reply:
updated